Vuologist/bumpie
# Huge PR (Adding About Us and FAQ that Peggy made, Routing, Footer/Header Logic)

Closes #200, closes #204, closes #205

Testing Instructions
1. Check if FAQ (/faq) and About US (/about-us) look like XD enough
2. Check routing on header works
3. Check header change vs logged in vs logged off

NOTE THERE IS A BUG IN THIS PR:
- WHEN NAVIGATING PAGES FOOTER WILL NOT CHANGE UNLESS PAGE FULLY REFRESHES, NEED TO OPEN ISSUE FOR THIS TO FIX
